specification/dartLangSpec.tex

```diff
@@ -2516,11 +2518,13 @@ \section{Classes}
 \rationale{%
 We want different behavior for concrete classes and abstract classes.
 If $A$ is intended to be abstract,
-we want the static checker to warn about any attempt to instantiate $A$,
+the static checker should report an error
+if an attempt is made to instantiate $A$,
 and we do not want the checker to complain about unimplemented methods in $A$.
 In contrast, if $A$ is intended to be concrete,
-the checker should warn about all unimplemented methods,
-but allow clients to instantiate it freely.%
+the checker should raise an error for all unimplemented methods,
+but clients should be allowed to instantiate it.
+This motivates the use of an explicit modifier to identify abstract classes.%
```

Comment:
I came across this old text which says that we'd 'warn' when an attempt is made to create an instance of an abstract class, and updated it to say that it is an error. This is unrelated but certainly needs to be fixed at some point.